src/screens/Signup/signup.css

```diff
@@ -0,0 +1,42 @@
+.signUp form {
+  width: 18.75rem;
+  margin: var(--size-8) auto;
+  border-radius: var(--size-1);
+  padding: var(--size-8);
+  border: 1px solid var(--wisdom-light);
+}
+
+.signUp input {
+  margin-bottom: var(--size-4);
+}
+
+.signUp .btn {
+  align-self: center;
+  width: 100%;
+}
+
+.signUp a {
```

Comment:
I guess this is an unused class, you may remove it

Reply:
fixed in https://github.com/saadahmadhere/Wisdom-library/pull/4/commits/59c52d8f239e025032b57404d0246a43d0ad4e20